examples/todomvc/src/lib.rs

```diff
@@ -28,7 +28,7 @@ pub fn run() -> Result<(), JsValue> {
     let document = utils::document();
     let container = document
         .query_selector(".todoapp")?
-        .ok_or_else(|| js_sys::Error::new("could not find `.todoapp` container"))?;
+        .ok_or_else(|| js_sys::Error::new("could not find .todoapp container"))?;
```

Comment:
Let's leave this as-is -- it isn't a template string because it is inside a normal string. The intention is markdown-esque formatting of a code snippet within prose.

Reply:
Got it !